app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -66,6 +68,13 @@ const CourseCreator = createReactClass({
       this.props.fetchCampaign(campaignParam);
     }
     this.props.fetchCoursesForUser(window.currentUser.id);
+    },
+
+  setCopyCourseAssignments(e) {
+    const scoped = e.target.checked;
```

Comment:
Naming this constant doesn't make the code any more readable. I suggest just using `e.target.checked` directly in the setState call.